### PR TITLE
WTAZ Site LC12 Closure

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Closing all entries associated with WTAZ location 12
- Datalogger and station entries remain for the borehole sensor at LC13
- Streams affected WTAZ LC 12 HH-ENZ and LH-ENZ

### CODE REVIEW

**Review by CBurton90 — APPROVED** [review]

File: install/sensors.csv [hunk 1]
```diff
@@ -854,7 +854,7 @@ Nanometrics,Trillium Compact 120PH-2,9335,MNTZ,10,0,,0,1.2,0,0,0,0,2022-12-19T22
 Nanometrics,Trillium Compact 120PH-2,9336,PAKZ,10,0,,0,1.2,0,0,0,0,2022-12-19T00:05:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9337,PTAZ,10,0,,0,1.2,0,0,0,0,2022-12-08T21:30:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9365,HTPZ,10,0,,0,1.2,0,0,0,0,2022-12-19T02:31:00Z,9999-01-01T00:00:00Z
-Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,12,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,9999-01-01T00:00:00Z
+Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,12,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,2023-12-13T22:03:00Z
```
Comment (CBurton90) [inline install/sensors.csv:857]: Sorry I think this is the wrong sensor. It should be a `20PH-2` (20s models rather than 120s)

**Review by CBurton90 — CHANGES_REQUESTED** [review]

**Review by CBurton90 — CHANGES_REQUESTED** [review]
@Thomas-Benson Looking at this PR I think I need to correct some things with @timjmcdougall from his work performed in June this year.

https://github.com/GeoNet/tickets/issues/13587

AFAIA SN `9366` was used to replace the borehole sensor on LC 13 and `8268` has remained at the surface on LC 12 since install. I will follow up to confirm.

**Comment by CBurton90** [comment]
@Thomas-Benson I can confirm the sensor removed this week was `Nanometrics,Trillium Compact 20PH-2,8268,WTAZ,12`. 

In June this year the borehole sensor (LC13) was swapped yet the surface sensor was not (LC12). Therefore the line:

`Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,12,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,2023-12-13T22:03:00Z`

needs changing to:

`Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,13,0,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,2023-12-13T22:03:00Z`

and LC12 for the 20PH-2 just need closing this week so:

`Nanometrics,Trillium Compact 20PH-2,8268,WTAZ,12,0,,0,0.5,0,0,0,0,2022-05-25T22:54:00Z,2023-12-13T22:03:00Z`

Sorry about this mess. Hopefully that makes sense to you.

**Comment by Thomas-Benson** [comment]
@CBurton90 Hopefully I have captured this correctly. My understanding is:
 - TC 120PH-2 9205 at LC 13 was removed in june
 - TC120PH-2 9366 was installed in June at LC 13 and remains in place.
 - TC20PH-2 8268 was left at LC 12 until being removed this week
The situation currenly in delta assumes that the sensor at LC 12 was swapped in June rather than the sensor at LC13

**Review by CBurton90 — APPROVED** [review]

**Comment by CBurton90** [comment]
@Thomas-Benson Yes exactly. Looks good to me now, thanks.

**Review by mnaguit — COMMENTED** [review]
This is really confusing. Doing my own check and just summarizing here for future reference (please correct me if I'm wrong):
- Back in June, the changes should have been `120PH-2` from `9205` to `9366` for `LC 13` (but the ticket says `20PH-2 (8268)`)
     - orientation for `120PH-2 (9205)` was updated from `36` to `138 `  --> should this change be for `9366`?

- Now in December, it is all about `20PH-2 (8268)` closing `LC 12`

Request:
- Can we have another round of check for azimuth and depth for all 3 sensors? @Thomas-Benson @CBurton90 @gva6500  Thanks!

**Comment by Thomas-Benson** [comment]
Good points @mnaguit, The depths and azimuths do need a bit more work, I'll get on to fixing them up. And we do have two different dates on which `20PH-2 (8268)` was meant to be removed 😕

**Comment by mnaguit** [comment]
Looks like the change in this PR for `20PH-2 (8268)` is already good.

**Comment by CBurton90** [comment]
I think the confusion is that the SN for the Trillium Compact removal in this ticket was wrong https://github.com/GeoNet/tickets/issues/13587.

It shows the TC-PH20 was removed when in fact the TC-PH120 was removed.

So for location code 13 the entries should be:

`Nanometrics,Trillium Compact 120PH-2,9205,WTAZ,13,36,,0,90,0,0,0,0,2023-04-12T03:28:00Z,2023-06-19T23:27:00Z`

`Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,13,138,,0,0.5,0,0,0,0,2023-06-20T00:00:00Z,9999-01-01T00:00:00Z`

I've added the correct azimuths.

Location code 12 looks correct to me, it remained the entire time.

cc @Thomas-Benson @mnaguit

**Comment by Thomas-Benson** [comment]
Added the (hopefully) correct azimuths and depths to the borehole deployments.

**Comment by mnaguit** [comment]
Hi @Thomas-Benson 
This is the entry for `9205`:

> Nanometrics,Trillium Compact 120PH-2,9205,WTAZ,13,36,,0,90,0,0,0,0,2023-04-12T03:28:00Z,2023-06-19T23:27:00Z

Thanks!

**Comment by CBurton90** [comment]
> Hi @Thomas-Benson This is the entry for `9205`:
> 
> > Nanometrics,Trillium Compact 120PH-2,9205,WTAZ,13,36,,0,90,0,0,0,0,2023-04-12T03:28:00Z,2023-06-19T23:27:00Z
> 
> Thanks!

@mnaguit That entry seems to be already present in delta so no changes required for that line.

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
> > Hi @Thomas-Benson This is the entry for `9205`:
> > > Nanometrics,Trillium Compact 120PH-2,9205,WTAZ,13,36,,0,90,0,0,0,0,2023-04-12T03:28:00Z,2023-06-19T23:27:00Z
> > 
> > 
> > Thanks!
> 
> @mnaguit That entry seems to be already present in delta so no changes required for that line.

Gotcha, the red highlight in this PR means that line is deleted. All good!

**Review by mnaguit — APPROVED** [review]

**Pull request merged**